UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Controllers/WebhookController.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using UKHO.MaritimeSafetyInformation.Common.Logging;
+
+namespace UKHO.MaritimeSafetyInformation.Web.Controllers
+{
+    public class WebhookController : BaseController<WebhookController>
+    {
+        public WebhookController(IHttpContextAccessor contextAccessor, ILogger<WebhookController> logger) : base(contextAccessor, logger)
+        {
+        }
+
+        [HttpOptions]
+        [Route("/webhook/newfilespublished")]
+        public IActionResult NewFilesPublishedOptions()
+        {
```

Comment:
Please add unit test cases for both options and webhook.

Reply:
Test cases added